src/App.tsx

```diff
@@ -15,36 +15,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+export const App = () => {
+  const [sortType, setSortType] = useState('default');
+  const [isReversed, setIsReversed] = useState(false);
+
+  const getSortedGoods = () => {
+    const sortedGoods = [...goodsFromServer];
+
+    if (sortType === 'alphabet') {
+      sortedGoods.sort((a, b) => a.localeCompare(b));
+    } else if (sortType === 'length') {
+      sortedGoods.sort((a, b) => a.length - b.length);
+    }
+
+    if (isReversed) {
+      sortedGoods.reverse();
+    }
+
+    return sortedGoods;
+  };
+
+  const handleSortAlphabetically = () => {
+    setSortType('alphabet');
+  };
+
+  const handleSortByLength = () => {
+    setSortType('length');
+  };
```

Comment:
[CODE STYLE] - Function's name should start with a verb (so you could easily tell what your method actually do); The name 'handleSortByLength' is good as it starts with a verb.